File: android/BertSpeak/app/src/main/java/chuckcoughlin/bert/service/DispatchServiceBinder.java
/**
 * Copyright 2019 Charles Coughlin. All rights reserved.
 * (MIT License)
 */
package chuckcoughlin.bert.service;

import android.os.Binder;

public class DispatchServiceBinder extends Binder {
    private DispatchService service;


    public DispatchServiceBinder(DispatchService s) {
        this.service = s;
    }
    public DispatchService getService() {
        return service;
    }

}

File: BertSpeak/root/src/main/kotlin/chuckcoughlin/bertspeak/service/DispatchServiceBinder.kt
/**
 * Copyright 2022-2023 Charles Coughlin. All rights reserved.
 * (MIT License)
 */
package chuckcoughlin.bertspeak.service

import android.os.Binder

class DispatchServiceBinder(private val dispatchService: DispatchService) : Binder() {
    val service = dispatchService
}
